Sources/Card/Models/OrdersV2Responses.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+#if canImport(PaymentsCore)
+import PaymentsCore
+#endif
+
+public struct PaymentSource: Decodable {
+
+    /// The card used as payment
+    public let card: PaymentSource.Card
+
+    public struct Card: Decodable {
+
+        /// The last four digits of the provided card
+        public let lastFourDigits: String?
+
+        /// The card network
+        /// - Examples: "VISA", "MASTERCARD"
+        public let brand: String?
+
+        /// The type of the provided card.
+        /// - Examples: "DEBIT", "CREDIT"
+        public let type: String?
+
+        /// The result of the 3DS challenge
+        public let authenticationResult: AuthenticationResult?
+    }
+}
+
+struct Link: Decodable {
```

Comment:
We should break these out into separate files and rename this file to `PaymentSource` to increase discoverability of these types in the project.